setup.sh

```diff
@@ -232,6 +235,12 @@ else
                 if [ $2 ] ; then
                         clean $2
                 else
+                        # For a fresh environment, move tfvars in order to force creation of new stack tfvars
+                        if [ -e "${ROOTPROJ}/stacks/${ENV}.tfvars" ] ; then
+                                mv ${ROOTPROJ}/stacks/${ENV}.tfvars ${ROOTPROJ}/stacks/${ENV}.tfvars.old
```

Comment:
Any particular reasons for moving it rather than just deleting it? Would people ever need to use/see the old version again?

Reply:
some people have settings in their stacks that they would like to keep, I will need to set up my own schedule for downscaling so want to copy those values across